src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -274,7 +276,7 @@ function FloatingActionButtonAndPopover(
                         text: translate('sidebarScreen.fabNewChat'),
                         onSelected: () => interceptAnonymousUser(Report.startNewChat),
                     },
-                    ...(canUseTrackExpense
+                    ...(canUseTrackExpense && selfDMReportID
```

Comment:
https://github.com/Expensify/App/assets/16502320/ae79be10-0607-4efc-8810-3c5b6a34986b

Reply:
> t will make the Track expense button disappear if it's not selfDMReportID 

You mean in case the selfDMReportID is undefined, the Track expense button will disappear, right?

Reply:
Yeah

Reply:
Checkout the video I posted.

Reply:
- When I tested the PR, I found out that there is a case where the selfDM is not loaded (as you mentioned), which leads to the "Track expense" button is not displayed. But after signing out, and signing in, the works well. 
- "The selfDM is not loaded" issue always appear on your side when testing the PR, or sometimes?

Reply:
Yeah it's also sometimes for me

Reply:
But logging out doesn't work for me either

https://github.com/Expensify/App/assets/16502320/301b3b96-692a-4ec9-a883-000d9dff4901

Reply:
- If it is sometimes, I think it is BE bug and I think BE needs to make sure selfDM is always returned once the user signs in. 
-  @thienlnam  When the selfDM data is not available, what should we do?
1. Hide the "Track expense" option (Contributor's suggestion)
2. Disable "Track expense" option (C+'s suggestion)

Reply:
I don't think this is BE bug, revert the change make it worked as expected

Reply:
OK, I see the problem now. For some accounts, ie my account, `selfDMReport` is not added to our `allReports`.

Reply:
Related slack thread:
https://expensify.slack.com/archives/C049HHMV9SM/p1711754709599749

Reply:
Yeah there definitely is a BE component to it - however I think we should also handle it better in the FE.

I like the idea of hiding the option

Reply:
Ok cool. I'll complete the checklist then